### DISCUSSION TITLE
questionable performance / page loading speed

### DISCUSSION THREAD

**Discussion by mluerig** [pradyunsg/furo#334] [Potential Issue] [answered]
I created some docs for my python package: https://www.phenopype.org/docs/index.html

I hadn't noticed while I was working on it, but it seems somewhat slow when switching between pages. I only noticed it when switching to light mode because the screen is black for a split-second. What could affect page buildup performance this much? Didn't do heavy customizing, I think...

**Comment by pradyunsg** [comment]
Not sure what's up with that. I'm away from laptops for the holidays, so it's unlikely I'd be able to take a closer look until late Jan/early Feb.

**Comment by pradyunsg** [comment, marked as answer]
Your logo is ~700 KB, which is way more than the rest of the page. That might be implicated somehow.
